README.md

```diff
@@ -60,18 +60,18 @@ All development for Smaily for Magento is [handled via GitHub](https://github.co
 1. Go to Stores -> Configuration -> Smaily email marketing and automation -> and click Module Configuration
 2. Open General Settings Tab
 3. Insert your Smaily API authentication information to get started and press Save Config
```

Comment:
I would rearrange some words to have a more logical flow to the sentence:

```suggestion
3. Insert your Smaily API authentication information and press Save Config to get started
```

Reply:
Updated readme in 0024f19dd392a59746405a5da136f3ed1f7480ee